# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/gamepad/ipc/GamepadEventChannelChild.cpp
# Commit: 8132dd939023
# Full Hash: 8132dd9390233328c8da7072f36859424a2b079d
# Author: Chris Martin <cmartin@mozilla.com>
# Date: 2020-10-14 21:42:48
# Regressor Bug: 1657404
# File Overlap Count: 2
# Description:
#   Bug 1657404 - Change PGamepadEventChannel to "refcounted protocol" r=handyman
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D93119
# ==============================================================================

diff -r 186ca9e93e1d -r 8132dd939023 dom/gamepad/ipc/GamepadEventChannelChild.cpp
--- a/dom/gamepad/ipc/GamepadEventChannelChild.cpp	Wed Oct 07 11:34:02 2020 +0000
+++ b/dom/gamepad/ipc/GamepadEventChannelChild.cpp	Wed Oct 14 15:58:36 2020 +0000
@@ -29,6 +29,11 @@
 
 }  // namespace
 
+already_AddRefed<GamepadEventChannelChild> GamepadEventChannelChild::Create() {
+  return RefPtr<GamepadEventChannelChild>(new GamepadEventChannelChild())
+      .forget();
+}
+
 mozilla::ipc::IPCResult GamepadEventChannelChild::RecvGamepadUpdate(
     const GamepadChangeEvent& aGamepadEvent) {
   DebugOnly<nsresult> rv =